src/webextension/list/manage/components/app.css

```diff
@@ -2,6 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+html {
+  box-sizing: border-box;
```

Comment:
I don't have a strong opinion either way on whether we should do this (though I lean softly towards not doing it as it makes our modular CSS more brittle), but I don't think now is the right time for it. The biggest issue is that this is only being applied in the management tab, so the `width`/`height` of things used in multiple views will be inconsistent (stuff in the `widgets/` dir is the most common example).

If we do want to change what box-sizing we use, it should be its own issue so that we cover everything and have an opportunity to compare things so that we can be sure there are no accidental visual changes.